### PR TITLE
Niko/aws config

### PR DESCRIPTION
**Pull request by niko79542**
Just added some settings for elastic beanstalk

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon",
+    "start": "node ./src/server/server.js",
```
Comment (jay-ithiel) [inline package.json:11]: Tried running `npm start` and it threw:

100x-backend@1.0.0 start /Users/jayhwang/.sandbox/projects/todo/100x/100x-Backend
node ./src/server/server.js

module.js:544
    throw err;
    ^

Error: Cannot find module 'dotenv'
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jayhwang/.sandbox/projects/todo/100x/100x-Backend/src/server/server.js:1:63)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)

File: src/config/config.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const config = {
   },
   production: {
     connection: { name: rdsConfig.RDS_DB_NAME, username: rdsConfig.RDS_USERNAME, password: rdsConfig.RDS_PASSWORD },
-    sequelizeOpts: { dialect: 'mysql', host: rdsConfig.RDS_HOSTNAME, port: rdsConfig.RDS_PORT, operatorsAliases: false }
+    sequelizeOpts: { dialect: 'mysql', host: rdsConfig.RDS_HOSTNAME, port: rdsConfig.RDS_PORT, operatorsAliases: false, maxConcurrentQueries: 100, pool: { maxConnections: 5, maxIdleTime: 30}, language: 'en' }
   },
```
Comment (jay-ithiel) [inline src/config/config.js:28]: the `rds.js` file we are pulling from isn't complete yet. Just made a todo commit in `rds.js`

**Review by jay-ithiel — CHANGES_REQUESTED** [review]
- npm start script can't find dotenv in server.js
- rds.js config isn't complete yet

**Comment by niko79542** [comment]
Sounds good.  dotenv was the npm package I was using to set environment variables.  Wasn't sure if theres a better way.  G2g for a bit.  Ill be back at 11:30

**Pull request closed without merging**